CHANGELOG.md

```diff
@@ -1,4 +1,25 @@
-## 2.8.1 (Unreleased)
+## 2.9.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.2.2047** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Added support for Designated Gateway in **aviatrix_gateway**
+2. Added support for enabling Encrypt EBS Volume in **aviatrix_gateway**
+3. Added Source NAT support of 'secondary' and 'custom' in **aviatrix_spoke_gateway**
+4. Added Destination NAT support in **aviatrix_gateway** and **aviatrix_spoke_gateway**
```

Comment:
1. Added support for "Designated Gateway" feature in **aviatrix_gateway**
2. Added support for encrypting the AWS EBS volume in **aviatrix_gateway**
3. Added support for "secondary" and "custom" Source NAT in **aviatrix_spoke_gateway**
4. Added support for Destination NAT in **aviatrix_gateway** and **aviatrix_spoke_gateway**